exercises/affine-cipher/README.md

```diff
@@ -0,0 +1,109 @@
+# Affine Cipher 
+
+Create an implementation of the affine cipher,
+an ancient encryption system created in the Middle East.
+ 
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with
+a mathematical function and then converted to the letter relating to
+its new numeric value. 
+
+To make an affine cipher, you must create two functions:
+
+1. `encrypt(message,a,b)`: A function to encrpt a message with keys `a` and `b`
+2. `decrypt(encryption,a,b)`: A function to decrypt an encrypted messsage with keys `a` and `b`
+
+## Algorithm 
+
+The algorithm is as below:
+ 
+** Note: `m` equals 26. You should take `m` as a constant. ** 
+ 
+1. the encryption function is:
+ 
+  `E(x) = (ax + b) mod m`
+  -  where `x` is the letter's index from 0 - length of alphabet - 1
```

Comment:
Only one of the `-` is a math minus here, correct? If so, please reword to avoid confusion.

Reply:
Agreed. I got this from the main problem-spec. It seems very unclear because of this will fix.